modules/nf-core/concoct/concoct/tests/main.nf.test.snap

```diff
@@ -0,0 +1,113 @@
+{
+    "sarscov2 - bam - stub": {
```

Comment:
This looks strange, with no md5sums

Reply:
Found it, everything shoud have md5sums now